dev/com.ibm.ws.persistence_fat/com.ibm.ws.persistence.consumer.testresource.bnd

```diff
@@ -21,7 +21,6 @@ Export-Package: \
 
 Import-Package: \
   javax.persistence;consumer="persistenceService",\
-  org.apache.commons.io*;version=1.4;packageType="was_internal",\
```

Comment:
do we know what the impact of this is?

Reply:
These were package "directives" that were built into our previous version of these commons libraries.  Because we are replacing with the latest public jar's these directives don't exist.